docs/uk/docs/index.md

```diff
@@ -116,7 +115,7 @@ $ pip install fastapi
 
 </div>
 
-You will also need an ASGI server, for production such as <a href="http://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> or <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
+Також Вам потрібен ASGI сервер для реального проекту, наприклад <a href="http://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> або <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```

Comment:
Don't think it's bad if we say production specifically instead of 'real project' even if the latter sounds slightly better

Reply:
```suggestion
Також Вам потрібен ASGI сервер для production етапу, наприклад <a href="http://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> або <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```